bndtools.core/_plugin.xml

```diff
@@ -605,9 +605,13 @@
 
                            <!--  Java element (src-folder, package or JUnit class) -->
                            <adapt type="org.eclipse.jdt.core.IJavaElement">
-                              <test property="org.eclipse.jdt.core.hasTypeOnClasspath" value="junit.framework.Test" />
-                              <test property="org.eclipse.jdt.junit.canLaunchAsJUnit" forcePluginActivation="true" />
                               <test property="org.bndtools.core.isInBndJavaProject" forcePluginActivation="true" />
+                              <test property="org.eclipse.jdt.junit.canLaunchAsJUnit" forcePluginActivation="true" />
+                              <or>
+                                 <test property="org.eclipse.jdt.core.hasTypeOnClasspath" value="org.junit.Test" />
+                                 <test property="org.eclipse.jdt.core.hasTypeOnClasspath" value="junit.framework.Test" />
+                                 <test property="org.eclipse.jdt.core.hasTypeOnClasspath" value="org.junit.jupiter.api.Test" />
```

Comment:
There are more Junit 5 types to consider. You may need to map this `<test>` to a set of `<tests>`:

https://github.com/bndtools/bnd/blob/9c99bbd913df0c2c78114868b61da4d3d50e0fea/maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/TestCases.java#L7-L11

Reply:
I don't think so. Please not this is not a check of the classes and how they are annotated, it's just checking if a class is on the classpath, Which means you only need a single one from each junit framework version.

Reply:
OK